lib/targetctl.js

```diff
@@ -26,6 +26,19 @@ function getWorkerInfo() {
   };
 }
 
+function queryCapabilities(req, rsp, callback) {
+  var worker = getWorkerInfo();
+
+  rsp.cmd = 'query-capabilities';
+  rsp.feature = req.feature;
+
+  if (rsp.feature === 'watchdog') {
+    rsp.supported = agent().internal.supports.watchdog;
```

Comment:
if its not supported, would be useful to have a .reasons, which would be an array of strings, each of which is a reason why its not supported, for example:

"watchdog not supported on this platform", and "addon not compiled", and "feature not licensed" (all three could be true at the same time). I can add the code to figure out all three, if you want.